lib/chef/chef_fs/data_handler/container_data_handler.rb

```diff
@@ -15,10 +15,10 @@ def preserve_key?(key)
           return key == "containername"
         end
 
-        def verify_integrity(object, entry, &on_error)
+        def verify_integrity(object, entry)
```

Comment:
I'm a bit hesitant about these ones. I personally like the `yield` keyword for non-logical reasons, but having the block argument makes the method signature more descriptive, especially for longer methods where the `yield` may be harder to spot on a quick reading. You can of course fix that with yard/rdoc/whatever but I don't know if we find that worthwhile.